plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/preferences/CloudSdkPreferenceArea.java

```diff
@@ -92,10 +95,18 @@ public void widgetSelected(SelectionEvent event) {
         WorkbenchUtil.openInBrowser(PlatformUI.getWorkbench(), event.text);
       }
     });
-    
-    sdkVersionLabel = new Label(parent, SWT.LEAD);
+
+    Composite versionArea = new Composite(parent, SWT.NONE);
+    sdkVersionLabel = new Label(versionArea, SWT.LEAD);
     sdkVersionLabel.setFont(contents.getFont());
     sdkVersionLabel.setText(Messages.getString("SdkVersion", "Unset")); //$NON-NLS-1$ //$NON-NLS-2$
+    updateSdk = new Button(versionArea, SWT.FLAT);
```

Comment:
Changing this to a normal `SWT.PUSH` button — I thought the `SWT.FLAT` would be smaller, but it's actually bigger than the other buttons.